fastly/block_fastly_service_v1_settings.go

```diff
@@ -86,13 +86,13 @@ func (h *SettingsServiceAttributeHandler) Register(s *schema.Resource) error {
 		Type:        schema.TypeBool,
 		Optional:    true,
 		Default:     false,
-		Description: "The default hostname",
+		Description: "Whether or not stale if error should be true",
 	}
 	s.Schema["stale_if_error_ttl"] = &schema.Schema{
 		Type:        schema.TypeInt,
 		Optional:    true,
 		Default:     43200,
-		Description: "The default hostname",
+		Description: "The TTL for stale if error",
```

Comment:
I would prefer us to use the same language as on the developer reference for consistency/accuracy:

> The default time-to-live (TTL) for serving the stale object for the version